src/inc/quic_platform_winuser.h

```diff
@@ -740,8 +740,44 @@ typedef struct CXPLAT_SQE {
 #if DEBUG
     BOOLEAN IsQueued; // Debug flag to catch double queueing.
 #endif
+    HANDLE WcpEvent;      // Manual-reset event for wake packets
+    HANDLE WaitCompletionPacket;    // Wait completion packet bound to Event
```

Comment:
There is no `Event`, this comment needs to be updated.

It would also be more helpful to have a comment about the "why" than the "what", explaining that these two handles control a wait completion packets allowing to safely wait until the IOCP is fully drained.